docs/mentoring.md

```diff
@@ -0,0 +1,44 @@
+# Mentoring
+
+To help you get up to speed, or further develop you Argo Workflows contribution skills, you can now get a mentor.
+
+Mentors will help you with things like:
+
+* Resources to learn Golang, Typescript and React.
+* Get Argo Workflows running in development mode on your laptop.
+* Understand key concepts and learn your way around the source code
+* Getting your first pull request with a code contribution created, reviewed and merged.
+
+Mentors and mentees must abide by the [code of conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). 
+
+## How To Get A Mentor
+
+Create a pull request which changes this page to add you to this list:
+
+Mentees:
+
+* Add your name here!
+
```

Comment:
Not sure this is the best process. I suggest better creating a Github issue for each mentee and making use of the Github projects Kanban board. Each mentor would have their own columnd and mentees would each be said issue.

Reply:
What kind of permissions do mentees need in order to manage the Kanban board?

Reply:
I choose this to be consistent with the list of mentors, and to record relationships. But, it does mean mentees need to create a PR, which they might not be able to do so when they need a mentor. I will change it